core-web/libs/portlets/dot-ema/src/lib/edit-ema-editor/components/edit-ema-palette/components/edit-ema-palette-contentlets/edit-ema-palette-contentlets.component.scss

```diff
@@ -0,0 +1,87 @@
+@use "variables" as *;
+
+:host {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+
+    ::ng-deep {
+        .p-paginator-icon {
+            color: $color-palette-primary;
+        }
+    }
+}
+
+.content-search {
+    display: flex;
+    align-items: center;
+    gap: $spacing-0;
+    top: 0;
+    padding: $spacing-3 $spacing-3;
+    position: sticky;
+    z-index: 99;
+    background: white;
+
+    .p-input-icon-right {
+        width: 100%;
+        input {
+            width: 100%;
+        }
+    }
+}
+
+.contentlets {
+    display: flex;
+    flex-direction: column;
+    padding: 0 $spacing-3;
+    height: 100%;
+
+    .contentlets-cards {
+        flex: 1;
+        overflow: scroll;
+    }
+
+    .contentlet-card {
+        display: grid;
+        grid-template-columns: max-content 3.37rem auto;
```

Comment:
Why don't you let the width of the thumbnail to drive this? Right now you have the width here an the height in the `dot-contentlet-thumbnail`

Reply:
Both height and width are accessories of this component, but internally they do not seem to be used

I reviewed several examples and, despite passing height and width, it is the container that gives it this size

What is being passed is the size of the icon (in case it does not have an image, the component places an icon)
![Screenshot 2024-01-02 at 13 55 12](https://github.com/dotCMS/core/assets/144152756/4c82975b-4bb0-4207-b2b2-afab0fb092f3)